rust/gui-client/src/about.ts

```diff
@@ -9,8 +7,8 @@ const gitVersionSpan = <HTMLSpanElement>document.getElementById("git-version");
 
 function get_cargo_version() {
   invoke("get_cargo_version")
-    .then((cargoVersion: string) => {
-      cargoVersionSpan.innerText = cargoVersion;
+    .then((cargoVersion: unknown) => {
```

Comment:
`xyz.then` is equivalent to `await xyz` in TS. Unfortunately not as chainable as in Rust but avoids the use of closures.